lms/djangoapps/certificates/docs/decisions/005-cert-display-settings.rst

```diff
@@ -0,0 +1,98 @@
+Certificate Display Settings
```

Comment:
Thanks for this ADR and the table. As someone who has only paid slight attention to the CAD/CDB work and the related changes, this will be super helpful.